text/0000-dumps.md

```diff
@@ -0,0 +1,120 @@
+- Title: Dumps
+- Start Date: 2022-01-12
+- UpdatedAt: 2022-01-12
+
+# Dumps
+
+## 1. Functional Specification
+
+### 1.1 Summary
+
+A dump is a compressed file containing an export of a MeiliSearch instance. It contains all indexes, documents, settings, tasks, and api keys but in a raw unprocessed form. A dump isn't an exact copy of a database—it is closer to a blueprint that allows to create an identical dataset.
+
+### 1.2 Motivation
+
+The dumps exists to upgrade from a previous version to a more recent version. It can also be a useful tool for loading a production state on a staging server to make changes and test them before propagating them to production.
+
+### 1.3 Explanations
+
+#### 1.3.1 Summary Key Points
+
+- A dump creation can be scheduled from the MeiliSearch API using the `POST - /dumps` endpoint.
+- A dump creation status can be tracked using the `GET - /dumps/{uid}/status` endpoint.
+- MeiliSearch can only create one dump at a time.
+- While a dump is in creation, the task queue is paused and no write operations can occur on the database.
```

Comment:
If I remember correctly what we did, while a dump is being created the task queue can't process any other updates, but it can still register new updates.
Do you confirm @MarinPostma?

Reply:
Yes it can register new updates while processing dump

Reply:
Thanks guys!

@meilisearch/docs-team "While a dump is processing, the task queue is paused and no write operations can occur on the database." We may want to delete this statement in the [documentation](https://docs.meilisearch.com/reference/api/dump.html#create-a-dump) 📡

Reply:
Hi @maryamsulemani97 

> When a dump is being imported, the task queue is paused and no write operations can occur on the database.

Yes, to be more clear, the task queue is not really **paused**, the API is not available at all. No **read or write** operations can be performed, except the import of the dump into the database at that time.

> When a dump is being created, the task queue can receive other future operations to perform but will not process any further tasks during dump creation, but will still accept new tasks in the queue.

Yes!